engine/explain_test.go

```diff
@@ -186,39 +187,83 @@ func TestAnalyzeOutputNode_Samples(t *testing.T) {
 	ctx := context.Background()
 
 	load := `load 30s
-				http_requests_total{pod="nginx-1"} 1+1x10
-				http_requests_total{pod="nginx-2"} 1+2x14`
+				http_requests_total{pod="nginx-1"} 1+1x100
+				http_requests_total{pod="nginx-2"} 1+1x100`
 
 	tstorage := promql.LoadedStorage(t, load)
 	defer tstorage.Close()
+	minT := tstorage.Head().Meta().MinTime
+	maxT := tstorage.Head().Meta().MaxTime
+
+	query, err := ng.NewInstantQuery(ctx, tstorage, nil, "http_requests_total", time.Unix(0, 0))
```

Comment:
What is the goal of this instant query? We don't seem to assert anything.

Reply:
THe assertions got lost during refactoring. Fixed it